README.md

```diff
@@ -1,14 +1,22 @@
-## graph-notebook
+## Welcome to the graph-notebook!
 
-Python package integrating jupyter notebooks with various graph-stores including
+
+The graph-notebook is an open-source Python package integrating jupyter notebooks with various graph-stores including
 [Apache TinkerPop](https://tinkerpop.apache.org/) and [RDF SPARQL](https://www.w3.org/TR/rdf-sparql-query/).
 
+<img align="right" width="550" src="images/graph-notebook-vis.png">
+
```

Comment:
Check the layout on this image and the text wrapping (see attached)
![Screen Shot 2020-11-09 at 8 26 05 PM](https://user-images.githubusercontent.com/8630663/98627833-fdfe3b80-22c9-11eb-9f80-dee997d5fe71.png)
 It's not rendering correctly for me.

Reply:
Why not reference this image the same way we reference other ones using the below syntax?

 \!\[my image](url-to-image) 

Is it because we are specifying height/width/alignment of the image?

Reply:
@beebs-systap FYI I don't see the same issue:
![image](https://user-images.githubusercontent.com/8711160/98698101-86142d80-232a-11eb-83c0-a61262148239.png)

Reply:
Perhaps it's not scaling properly with different resolutions? Can we try it without text wrapping?

Reply:
The link from `additional_databases` to that folder got lost as part of this PR I think (in the Introduction section)